docs/quick_start.md

```diff
@@ -29,6 +29,43 @@ Paste the following in a macOS Terminal and press enter to install Homebrew.
 Note: *When this command is run, additional installations may be automatically added by Homebrew based on your machine's configurations.
 Homebrew will prompt you to `Press RETURN to continue or any other key to abort`. Please continue with these installations by pressing `Return`*
 
+### Start your new task from our template repository
```

Comment:
I would actually put this first? 

1) Copy repo
2) Clone and move to repo
3) Install prerequisites

Reply:
isn't it the same existing order, could you clarify what step is different?